### PR TITLE
fix bad repair blob code

### PR DESCRIPTION
**Pull request by bunniewormy**
## Status

- **READY**: this PR is (to the best of your knowledge) ready to be incorporated into the game.

## Description

code was pepega

### CODE REVIEW

**Comment by mugg91** [comment]
Not completely fixed yet, see pm.

**Comment by bunniewormy** [comment]
pushed a fix

File: Entities/Common/Building/BlobPlacement.as [hunk 1]
```diff
@@ -8,12 +8,12 @@
 #include "Requirements.as"
 #include "RunnerTextures.as"
 
-void PlaceBlob(CBlob@ this, CBlob @blob, Vec2f cursorPos, bool repairing=false, CBlob@ repairBlob=null)
+bool PlaceBlob(CBlob@ this, CBlob @blob, Vec2f cursorPos, bool repairing = false, CBlob@ repairBlob = null)
```
Comment (AsuMagic) [inline Entities/Common/Building/BlobPlacement.as:11]: (FWIW, it seems to me that `repairing` is equivalent to `repairBlob !is null` and is not necessary anymore?)

**Review by AsuMagic — APPROVED** [review]

**Comment by mugg91** [comment]
I can't review anything for the next 18 hours, just in case you want to release a hotfix before then.

**Comment by mugg91** [comment]
I'm here now. Looks like there are no issues.

**Comment by mugg91** [comment]
I did find a problem now. See pm. Will require another hotfix.

**Comment by mugg91** [comment]
Looks like said problem was fixed by PR #1452 that got merged but didn't go live yet.

**Pull request merged**